components/Map.vue

```diff
@@ -65,62 +71,105 @@ export default {
   },
   methods: {
     idle() {
```

Comment:
Not related to this specific pull request, but out of interest... when does the idle method get called?  Do we set an interval somewhere ourselves or is it a special Vue method?